src/TRoom.cpp

```diff
@@ -101,7 +101,7 @@ QString TRoom::dirCodeToDisplayName(const int dirCode) const
     }
 }
 
-QString TRoom::dirCodeToString(const int dirCode) const
+/*static*/ QString TRoom::dirCodeToString(const int dirCode)
```

Comment:
I only picked up on marking with this comment such methods in this manner recently but I think it does help as a reminder when one is doing a global search on the use of the method's name! :nerd_face: